Assets/Scripts/Audio/PlayerSFX.cs

```diff
@@ -98,15 +97,25 @@ private void Dodge()
     {
         int i = Random.Range(0, grassRoll.Length);
         int j = Random.Range(0, armourJingle.Length);
-        audioPlayer.PlayOnce(armourJingle[j], audioManager.SFXVol/2);
-        audioPlayer.PlayOnce(rollArray[i], audioManager.SFXVol);
-        //Debug.Log(i);
+        
+        if (armourJingle.Length > i && armourJingle[i] != null)
```

Comment:
Just going for the simple solution here while we aim for stability, but we should really refactor this audio system later if we get the chance